src/components/settings/SettingsLMSTab/index.jsx

```diff
@@ -111,6 +125,7 @@ export default function SettingsLMSTab(enterpriseId) {
   useEffect(() => {
     // On load fetch potential existing configs
     fetchExistingConfigs();
+    checkIfCustomerNeedsIDPConfig();
```

Comment:
I also think you can directly populate this value as a boolean in the state.portalConfiguration and then just read it in this component just like the enterpriseId, it will also avoid the extra re-render you will have with this check.

The only disadvantage of this method is that you will need to reload page if the enterprise customer state is changed while the UI is being used. I don't know if that is an issue at all honestly but if it is, you can have that portalConfiguration be refetched instead ... to apply the change (if this is even an issue). Arguably that issue exists for all values used in the enterprise configuration anyway